doc/code_style/mom.cpp

```diff
@@ -171,7 +163,7 @@ void YourMom::print_mass() {
 bool YourMom::tick() {
 	// Note the spaces!
 	//|                | |  |
-	//v                v v  v
+	// v                v v  v
```

Comment:
this should stay, but of course that's only important for this style file, probably all other comments should have a space after `//`